src/components/opt-in-out/index.tsx

```diff
@@ -76,29 +53,12 @@ export default function OptInOut({ platform, redirectPath }: OptInOutProps) {
     if (typeof optInFrom == 'string' && isOptInPlatformOption(optInFrom)) {
       makeBetaWelcomeToast(optInFrom)
     }
-  }, [optInFrom, handleOptOut])
```

Comment:
This was a seemingly unnecessary dep, as this function isn't used in the toast generation.